tests/modules/transformer/transformer_embeddings_test.py

```diff
@@ -1,50 +1,167 @@
+import pytest
 import copy
 import torch
+from torch.testing import assert_allclose
 
-from allennlp.common import Params
-from allennlp.modules.transformer import TransformerEmbeddings
+from allennlp.common import Params, FromParams
+from allennlp.common import cached_transformers
+
+from transformers.models.bert.configuration_bert import BertConfig
+from transformers.models.bert.modeling_bert import BertEmbeddings
+from transformers.models.albert.configuration_albert import AlbertConfig
+from transformers.models.albert.modeling_albert import AlbertEmbeddings
+
+from allennlp.common.testing import assert_equal_parameters
+from allennlp.modules.transformer import (
+    TransformerEmbeddings,
+    ImageFeatureEmbeddings,
+    TransformerModule,
+)
 from allennlp.common.testing import AllenNlpTestCase
 
+PARAMS_DICT = {
+    "vocab_size": 20,
+    "embedding_size": 5,
+    "pad_token_id": 0,
+    "max_position_embeddings": 3,
+    "type_vocab_size": 2,
+    "dropout": 0.5,
+}
+
+
+def get_modules(params_dict):
+    modules = {}
+    params = copy.deepcopy(params_dict)
+
+    params["hidden_dropout_prob"] = params.pop("dropout")
+    params["hidden_size"] = params.pop("embedding_size")
+
+    # bert, roberta, electra self attentions have the same code.
+
+    torch.manual_seed(1234)
+    hf_module = BertEmbeddings(BertConfig(**params))
+    modules["bert"] = hf_module
+
+    albertparams = copy.deepcopy(params_dict)
+    albertparams["hidden_dropout_prob"] = albertparams.pop("dropout")
+
+    torch.manual_seed(1234)
+    hf_module = AlbertEmbeddings(AlbertConfig(**albertparams))
```

Comment:
Looks like it's tested after all, so Codecov is just wrong?

If that's the case we should probably turn it off.